packages/@ember/-internals/routing/lib/services/router.ts

```diff
@@ -474,6 +476,39 @@ export default class RouterService extends Service {
   */
 }
 
+if (EMBER_ROUTING_ROUTER_SERVICE_REFRESH) {
+  RouterService.reopen({
+    /**
+     * Refreshes all currently active routes, doing a full transition.
+     * If a route name is provided and refers to a currently active route,
+     * it will refresh only that route and its descendents.
+     * Returns a promise that will be resolved once the refresh is complete.
+     * All resetController, beforeModel, model, afterModel, redirect, and setupController
+     * hooks will be called again. You will get new data from the model hook.
+     *
+     * @method refresh
+     * @param {String} [pivotRouteName] the route to refresh (along with all child routes)
```

Comment:
I think you are right, using `routeName` here seems better. I personally think that internally referring to it as a "pivot route" is good (like the internal local variable name) to infer the  "the specified route and its descendents" thing without having to say it everywhere.

Reply:
So, does that mean I change it back to `routeName`? :smile:

Reply:
Haha, sorry yes. I'd like to document the param as `routeName` here.